packages/RatPack.package/RPApplication.class/instance/initializeRoutes.st

```diff
@@ -3,4 +3,8 @@ initializeRoutes
 
 	routes := IdentityDictionary new.
 	self initializePragmaRoutes.
-	self initializeEndpointRoutes 	
\ No newline at end of file
+	self initializeEndpointRoutes. 	
+		
+	self defaultRoutesModelLookupTable: Dictionary new.
```

Comment:
Doesn't initializeEndpointRoutes access this dictionary and write entries to it? Why is this correct?

Reply:
This is used to map route name to model name so we can access models which have different names than their routes. `initializeEndpointRoutes` does not access it

Reply:
Ah I see